docs/sources/flow/getting-started/configure-agent-clustering.md

```diff
@@ -17,21 +17,21 @@ individual agents can work together for workload distribution and high
 availability.
 
 {{% admonition type="note" %}}
-Clustering is a [beta]({{< relref "../../stability.md#beta" >}}) feature. Beta features are subject to breaking
-changes and may be replaced with equivalent functionality that covers the same
-use case.
+Clustering is a [beta](http://www.grafana.com/docs/agent/flow/stability.md#beta) feature.
```

Comment:
nit: did you want this link stored down below? I'm just sanity checking the new pattern I'm seeing. I'm good either way.

Reply:
I don't think this link is working either. I tried it out myself in `make docs`

Reply:
Oh my I messed that one up. It's missing a bunch of the URL. I'l fix it in a new commit